lib/ansible/config/base.yml

```diff
@@ -36,15 +36,29 @@ ANSIBLE_COW_SELECTION:
   env: [{name: ANSIBLE_COW_SELECTION}]
   ini:
   - {key: cow_selection, section: defaults}
-ANSIBLE_COW_WHITELIST:
-  name: Cowsay filter whitelist
+ANSIBLE_COW_ENABLED:
```

Comment:
Bikeshed: `ANSIBLE_COW_ACCEPTLIST` would be clearer that it's a list, not a boolean